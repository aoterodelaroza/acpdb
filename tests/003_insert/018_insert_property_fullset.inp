## check: 018_insert_property_fullset.out -a1e-10
## delete: 018_insert_property_fullset.db
## labels: regression quick

verbose
system rm -f 018_insert_property_fullset.db
connect 018_insert_property_fullset.db

insert litref bibtex ../zz_source/bsip2.bib
insert method b3lyp/6-31+G*
 litrefs bsip1 boys1970
 description This is a method.
end
insert method test
end
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method b3lyp/6-31+G*
end
insert property strange-
 property_type dipole
 set s22
end

print property
