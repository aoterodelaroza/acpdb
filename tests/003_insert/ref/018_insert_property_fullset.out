%% verbose
%% system rm -f 018_insert_property_fullset.db
* SYSTEM: rm -f 018_insert_property_fullset.db

%% connect 018_insert_property_fullset.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 018_insert_property_fullset.db
Creating skeleton database 

%% insert litref bibtex ../zz_source/bsip2.bib
* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

%% insert method b3lyp/6-31+G*
* INSERT: insert data into the database (METHOD)
# INSERT METHOD b3lyp/6-31+G*

%% insert method test
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

%% insert set s22
* INSERT: insert data into the database (SET)
# INSERT SET s22
# INSERT STRUCTURE s22.nh3_nh3
# INSERT STRUCTURE s22.nh3_nh3_1
# INSERT STRUCTURE s22.nh3_nh3_2
# INSERT PROPERTY s22.nh3_nh3
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.nh3_nh3)
# INSERT STRUCTURE s22.h2o_h2o
# INSERT STRUCTURE s22.h2o_h2o_1
# INSERT STRUCTURE s22.h2o_h2o_2
# INSERT PROPERTY s22.h2o_h2o
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.h2o_h2o)
# INSERT STRUCTURE s22.h2co2_h2co2
# INSERT STRUCTURE s22.h2co2_h2co2_1
# INSERT STRUCTURE s22.h2co2_h2co2_2
# INSERT PROPERTY s22.h2co2_h2co2
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.h2co2_h2co2)
# INSERT STRUCTURE s22.formamide_formamide
# INSERT STRUCTURE s22.formamide_formamide_1
# INSERT STRUCTURE s22.formamide_formamide_2
# INSERT PROPERTY s22.formamide_formamide
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.formamide_formamide)
# INSERT STRUCTURE s22.uracil_uracil_hb
# INSERT STRUCTURE s22.uracil_uracil_hb_1
# INSERT STRUCTURE s22.uracil_uracil_hb_2
# INSERT PROPERTY s22.uracil_uracil_hb
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.uracil_uracil_hb)
# INSERT STRUCTURE s22.pyridoxine_aminopyridine
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_1
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY s22.pyridoxine_aminopyridine
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.pyridoxine_aminopyridine)
# INSERT STRUCTURE s22.adenine_thymine_wcc1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_2
# INSERT PROPERTY s22.adenine_thymine_wcc1
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.adenine_thymine_wcc1)
# INSERT STRUCTURE s22.ch4_ch4
# INSERT STRUCTURE s22.ch4_ch4_1
# INSERT STRUCTURE s22.ch4_ch4_2
# INSERT PROPERTY s22.ch4_ch4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.ch4_ch4)
# INSERT STRUCTURE s22.c2h4_c2h4
# INSERT STRUCTURE s22.c2h4_c2h4_1
# INSERT STRUCTURE s22.c2h4_c2h4_2
# INSERT PROPERTY s22.c2h4_c2h4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c2h4_c2h4)
# INSERT STRUCTURE s22.c6h6_ch4
# INSERT STRUCTURE s22.c6h6_ch4_1
# INSERT STRUCTURE s22.c6h6_ch4_2
# INSERT PROPERTY s22.c6h6_ch4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_ch4)
# INSERT STRUCTURE s22.c6h6_c6h6_pd
# INSERT STRUCTURE s22.c6h6_c6h6_pd_1
# INSERT STRUCTURE s22.c6h6_c6h6_pd_2
# INSERT PROPERTY s22.c6h6_c6h6_pd
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_c6h6_pd)
# INSERT STRUCTURE s22.pyrazine_pyrazine
# INSERT STRUCTURE s22.pyrazine_pyrazine_1
# INSERT STRUCTURE s22.pyrazine_pyrazine_2
# INSERT PROPERTY s22.pyrazine_pyrazine
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.pyrazine_pyrazine)
# INSERT STRUCTURE s22.uracil_uracil_stack
# INSERT STRUCTURE s22.uracil_uracil_stack_1
# INSERT STRUCTURE s22.uracil_uracil_stack_2
# INSERT PROPERTY s22.uracil_uracil_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.uracil_uracil_stack)
# INSERT STRUCTURE s22.indole_c6h6_stack
# INSERT STRUCTURE s22.indole_c6h6_stack_1
# INSERT STRUCTURE s22.indole_c6h6_stack_2
# INSERT PROPERTY s22.indole_c6h6_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.indole_c6h6_stack)
# INSERT STRUCTURE s22.adenine_thymine_stack
# INSERT STRUCTURE s22.adenine_thymine_stack_1
# INSERT STRUCTURE s22.adenine_thymine_stack_2
# INSERT PROPERTY s22.adenine_thymine_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.adenine_thymine_stack)
# INSERT STRUCTURE s22.c2h4_c2h2
# INSERT STRUCTURE s22.c2h4_c2h2_1
# INSERT STRUCTURE s22.c2h4_c2h2_2
# INSERT PROPERTY s22.c2h4_c2h2
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c2h4_c2h2)
# INSERT STRUCTURE s22.c6h6_h2o
# INSERT STRUCTURE s22.c6h6_h2o_1
# INSERT STRUCTURE s22.c6h6_h2o_2
# INSERT PROPERTY s22.c6h6_h2o
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_h2o)
# INSERT STRUCTURE s22.c6h6_nh3
# INSERT STRUCTURE s22.c6h6_nh3_1
# INSERT STRUCTURE s22.c6h6_nh3_2
# INSERT PROPERTY s22.c6h6_nh3
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_nh3)
# INSERT STRUCTURE s22.c6h6_hcn
# INSERT STRUCTURE s22.c6h6_hcn_1
# INSERT STRUCTURE s22.c6h6_hcn_2
# INSERT PROPERTY s22.c6h6_hcn
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_hcn)
# INSERT STRUCTURE s22.c6h6_c6h6_t
# INSERT STRUCTURE s22.c6h6_c6h6_t_1
# INSERT STRUCTURE s22.c6h6_c6h6_t_2
# INSERT PROPERTY s22.c6h6_c6h6_t
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_c6h6_t)
# INSERT STRUCTURE s22.indole_c6h6_t
# INSERT STRUCTURE s22.indole_c6h6_t_1
# INSERT STRUCTURE s22.indole_c6h6_t_2
# INSERT PROPERTY s22.indole_c6h6_t
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.indole_c6h6_t)
# INSERT STRUCTURE s22.phenol_phenol
# INSERT STRUCTURE s22.phenol_phenol_1
# INSERT STRUCTURE s22.phenol_phenol_2
# INSERT PROPERTY s22.phenol_phenol
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.phenol_phenol)

%% insert property strange-
* INSERT: insert data into the database (PROPERTY)
# INSERT PROPERTY strange-s22.nh3_nh3
# INSERT PROPERTY strange-s22.nh3_nh3_1
# INSERT PROPERTY strange-s22.nh3_nh3_2
# INSERT PROPERTY strange-s22.h2o_h2o
# INSERT PROPERTY strange-s22.h2o_h2o_1
# INSERT PROPERTY strange-s22.h2o_h2o_2
# INSERT PROPERTY strange-s22.h2co2_h2co2
# INSERT PROPERTY strange-s22.h2co2_h2co2_1
# INSERT PROPERTY strange-s22.h2co2_h2co2_2
# INSERT PROPERTY strange-s22.formamide_formamide
# INSERT PROPERTY strange-s22.formamide_formamide_1
# INSERT PROPERTY strange-s22.formamide_formamide_2
# INSERT PROPERTY strange-s22.uracil_uracil_hb
# INSERT PROPERTY strange-s22.uracil_uracil_hb_1
# INSERT PROPERTY strange-s22.uracil_uracil_hb_2
# INSERT PROPERTY strange-s22.pyridoxine_aminopyridine
# INSERT PROPERTY strange-s22.pyridoxine_aminopyridine_1
# INSERT PROPERTY strange-s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY strange-s22.adenine_thymine_wcc1
# INSERT PROPERTY strange-s22.adenine_thymine_wcc1_1
# INSERT PROPERTY strange-s22.adenine_thymine_wcc1_2
# INSERT PROPERTY strange-s22.ch4_ch4
# INSERT PROPERTY strange-s22.ch4_ch4_1
# INSERT PROPERTY strange-s22.ch4_ch4_2
# INSERT PROPERTY strange-s22.c2h4_c2h4
# INSERT PROPERTY strange-s22.c2h4_c2h4_1
# INSERT PROPERTY strange-s22.c2h4_c2h4_2
# INSERT PROPERTY strange-s22.c6h6_ch4
# INSERT PROPERTY strange-s22.c6h6_ch4_1
# INSERT PROPERTY strange-s22.c6h6_ch4_2
# INSERT PROPERTY strange-s22.c6h6_c6h6_pd
# INSERT PROPERTY strange-s22.c6h6_c6h6_pd_1
# INSERT PROPERTY strange-s22.c6h6_c6h6_pd_2
# INSERT PROPERTY strange-s22.pyrazine_pyrazine
# INSERT PROPERTY strange-s22.pyrazine_pyrazine_1
# INSERT PROPERTY strange-s22.pyrazine_pyrazine_2
# INSERT PROPERTY strange-s22.uracil_uracil_stack
# INSERT PROPERTY strange-s22.uracil_uracil_stack_1
# INSERT PROPERTY strange-s22.uracil_uracil_stack_2
# INSERT PROPERTY strange-s22.indole_c6h6_stack
# INSERT PROPERTY strange-s22.indole_c6h6_stack_1
# INSERT PROPERTY strange-s22.indole_c6h6_stack_2
# INSERT PROPERTY strange-s22.adenine_thymine_stack
# INSERT PROPERTY strange-s22.adenine_thymine_stack_1
# INSERT PROPERTY strange-s22.adenine_thymine_stack_2
# INSERT PROPERTY strange-s22.c2h4_c2h2
# INSERT PROPERTY strange-s22.c2h4_c2h2_1
# INSERT PROPERTY strange-s22.c2h4_c2h2_2
# INSERT PROPERTY strange-s22.c6h6_h2o
# INSERT PROPERTY strange-s22.c6h6_h2o_1
# INSERT PROPERTY strange-s22.c6h6_h2o_2
# INSERT PROPERTY strange-s22.c6h6_nh3
# INSERT PROPERTY strange-s22.c6h6_nh3_1
# INSERT PROPERTY strange-s22.c6h6_nh3_2
# INSERT PROPERTY strange-s22.c6h6_hcn
# INSERT PROPERTY strange-s22.c6h6_hcn_1
# INSERT PROPERTY strange-s22.c6h6_hcn_2
# INSERT PROPERTY strange-s22.c6h6_c6h6_t
# INSERT PROPERTY strange-s22.c6h6_c6h6_t_1
# INSERT PROPERTY strange-s22.c6h6_c6h6_t_2
# INSERT PROPERTY strange-s22.indole_c6h6_t
# INSERT PROPERTY strange-s22.indole_c6h6_t_1
# INSERT PROPERTY strange-s22.indole_c6h6_t_2
# INSERT PROPERTY strange-s22.phenol_phenol
# INSERT PROPERTY strange-s22.phenol_phenol_1
# INSERT PROPERTY strange-s22.phenol_phenol_2

%% print property
* PRINT: print the contents of the database 

| id| key| property_type| setid| orderid| nstructures|
| 1| s22.nh3_nh3| 1| 1| 1| 3|
| 2| s22.h2o_h2o| 1| 1| 2| 3|
| 3| s22.h2co2_h2co2| 1| 1| 3| 3|
| 4| s22.formamide_formamide| 1| 1| 4| 3|
| 5| s22.uracil_uracil_hb| 1| 1| 5| 3|
| 6| s22.pyridoxine_aminopyridine| 1| 1| 6| 3|
| 7| s22.adenine_thymine_wcc1| 1| 1| 7| 3|
| 8| s22.ch4_ch4| 1| 1| 8| 3|
| 9| s22.c2h4_c2h4| 1| 1| 9| 3|
| 10| s22.c6h6_ch4| 1| 1| 10| 3|
| 11| s22.c6h6_c6h6_pd| 1| 1| 11| 3|
| 12| s22.pyrazine_pyrazine| 1| 1| 12| 3|
| 13| s22.uracil_uracil_stack| 1| 1| 13| 3|
| 14| s22.indole_c6h6_stack| 1| 1| 14| 3|
| 15| s22.adenine_thymine_stack| 1| 1| 15| 3|
| 16| s22.c2h4_c2h2| 1| 1| 16| 3|
| 17| s22.c6h6_h2o| 1| 1| 17| 3|
| 18| s22.c6h6_nh3| 1| 1| 18| 3|
| 19| s22.c6h6_hcn| 1| 1| 19| 3|
| 20| s22.c6h6_c6h6_t| 1| 1| 20| 3|
| 21| s22.indole_c6h6_t| 1| 1| 21| 3|
| 22| s22.phenol_phenol| 1| 1| 22| 3|
| 23| strange-s22.nh3_nh3| 3| 1| 1| 1|
| 24| strange-s22.nh3_nh3_1| 3| 1| 2| 1|
| 25| strange-s22.nh3_nh3_2| 3| 1| 3| 1|
| 26| strange-s22.h2o_h2o| 3| 1| 4| 1|
| 27| strange-s22.h2o_h2o_1| 3| 1| 5| 1|
| 28| strange-s22.h2o_h2o_2| 3| 1| 6| 1|
| 29| strange-s22.h2co2_h2co2| 3| 1| 7| 1|
| 30| strange-s22.h2co2_h2co2_1| 3| 1| 8| 1|
| 31| strange-s22.h2co2_h2co2_2| 3| 1| 9| 1|
| 32| strange-s22.formamide_formamide| 3| 1| 10| 1|
| 33| strange-s22.formamide_formamide_1| 3| 1| 11| 1|
| 34| strange-s22.formamide_formamide_2| 3| 1| 12| 1|
| 35| strange-s22.uracil_uracil_hb| 3| 1| 13| 1|
| 36| strange-s22.uracil_uracil_hb_1| 3| 1| 14| 1|
| 37| strange-s22.uracil_uracil_hb_2| 3| 1| 15| 1|
| 38| strange-s22.pyridoxine_aminopyridine| 3| 1| 16| 1|
| 39| strange-s22.pyridoxine_aminopyridine_1| 3| 1| 17| 1|
| 40| strange-s22.pyridoxine_aminopyridine_2| 3| 1| 18| 1|
| 41| strange-s22.adenine_thymine_wcc1| 3| 1| 19| 1|
| 42| strange-s22.adenine_thymine_wcc1_1| 3| 1| 20| 1|
| 43| strange-s22.adenine_thymine_wcc1_2| 3| 1| 21| 1|
| 44| strange-s22.ch4_ch4| 3| 1| 22| 1|
| 45| strange-s22.ch4_ch4_1| 3| 1| 23| 1|
| 46| strange-s22.ch4_ch4_2| 3| 1| 24| 1|
| 47| strange-s22.c2h4_c2h4| 3| 1| 25| 1|
| 48| strange-s22.c2h4_c2h4_1| 3| 1| 26| 1|
| 49| strange-s22.c2h4_c2h4_2| 3| 1| 27| 1|
| 50| strange-s22.c6h6_ch4| 3| 1| 28| 1|
| 51| strange-s22.c6h6_ch4_1| 3| 1| 29| 1|
| 52| strange-s22.c6h6_ch4_2| 3| 1| 30| 1|
| 53| strange-s22.c6h6_c6h6_pd| 3| 1| 31| 1|
| 54| strange-s22.c6h6_c6h6_pd_1| 3| 1| 32| 1|
| 55| strange-s22.c6h6_c6h6_pd_2| 3| 1| 33| 1|
| 56| strange-s22.pyrazine_pyrazine| 3| 1| 34| 1|
| 57| strange-s22.pyrazine_pyrazine_1| 3| 1| 35| 1|
| 58| strange-s22.pyrazine_pyrazine_2| 3| 1| 36| 1|
| 59| strange-s22.uracil_uracil_stack| 3| 1| 37| 1|
| 60| strange-s22.uracil_uracil_stack_1| 3| 1| 38| 1|
| 61| strange-s22.uracil_uracil_stack_2| 3| 1| 39| 1|
| 62| strange-s22.indole_c6h6_stack| 3| 1| 40| 1|
| 63| strange-s22.indole_c6h6_stack_1| 3| 1| 41| 1|
| 64| strange-s22.indole_c6h6_stack_2| 3| 1| 42| 1|
| 65| strange-s22.adenine_thymine_stack| 3| 1| 43| 1|
| 66| strange-s22.adenine_thymine_stack_1| 3| 1| 44| 1|
| 67| strange-s22.adenine_thymine_stack_2| 3| 1| 45| 1|
| 68| strange-s22.c2h4_c2h2| 3| 1| 46| 1|
| 69| strange-s22.c2h4_c2h2_1| 3| 1| 47| 1|
| 70| strange-s22.c2h4_c2h2_2| 3| 1| 48| 1|
| 71| strange-s22.c6h6_h2o| 3| 1| 49| 1|
| 72| strange-s22.c6h6_h2o_1| 3| 1| 50| 1|
| 73| strange-s22.c6h6_h2o_2| 3| 1| 51| 1|
| 74| strange-s22.c6h6_nh3| 3| 1| 52| 1|
| 75| strange-s22.c6h6_nh3_1| 3| 1| 53| 1|
| 76| strange-s22.c6h6_nh3_2| 3| 1| 54| 1|
| 77| strange-s22.c6h6_hcn| 3| 1| 55| 1|
| 78| strange-s22.c6h6_hcn_1| 3| 1| 56| 1|
| 79| strange-s22.c6h6_hcn_2| 3| 1| 57| 1|
| 80| strange-s22.c6h6_c6h6_t| 3| 1| 58| 1|
| 81| strange-s22.c6h6_c6h6_t_1| 3| 1| 59| 1|
| 82| strange-s22.c6h6_c6h6_t_2| 3| 1| 60| 1|
| 83| strange-s22.indole_c6h6_t| 3| 1| 61| 1|
| 84| strange-s22.indole_c6h6_t_1| 3| 1| 62| 1|
| 85| strange-s22.indole_c6h6_t_2| 3| 1| 63| 1|
| 86| strange-s22.phenol_phenol| 3| 1| 64| 1|
| 87| strange-s22.phenol_phenol_1| 3| 1| 65| 1|
| 88| strange-s22.phenol_phenol_2| 3| 1| 66| 1|

